CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ IMPROVEMENTS:
 BUG FIXES:
 
 * secrets/gcp: Ensure that the IAM policy version is appropriately set after a roleset's bindings have changed. [[GH-9603](https://github.com/hashicorp/vault/pull/9603)]
+* secrets/openldap: Add "ad" schema that allows the engine to correctly rotate AD passwords.
```

Comment:
This actually hasn't been added to Vault yet, right?  IE go.mod wasn't updated.

Reply:
Good point, I've moved it to the "next" section.